### DISCUSSION TITLE
Loading a model file at runtime

### DISCUSSION THREAD

**Discussion by ghost** [stride3d/stride#1355] [Q&A] [answered]
is it possible to load models from a folder on disk, without building the models into the project? i'm experimenting with a networking library and i want to have the server send an object to the client that the client then loads, without the client knowing what the object is prior to the connection. is that possible? to start with, i want to know if it's possible to dynamically load an object from disk.

**Comment by Eideren** [comment, marked as answer]
See point 5 in https://doc.stride3d.net/latest/en/manual/scripts/create-a-model-from-code.html#create-a-procedural-model

**Reply by ghost** [reply]
i should have been more specific, i currently have an io stream pointing to the model i want to load, but i'm unsure of how to convert that into a model. i will take another look at the procedural model documentation though.

**Reply by Eideren** [reply]
Depends on the model format, use one of the libraries out there to read it, [assimp](https://github.com/assimp/assimp) for example, and move the vertex data from that api to stride's
